package.json

```diff
@@ -105,5 +105,8 @@
     "validate.js": "^0.13.1",
     "virtual-dom": "^2.1.1",
     "yaml-loader": "^0.6.0"
+  },
+  "resolutions": {
+    "jquery-ui/jquery": "3.7.0"
```

Comment:
What is this `resolutions` about?

Reply:
`jquery-ui` depends on `jquery >= 1 && < 4.0.0`. We depend on `jquery ^3.7.0`. That means we install 3.7.1 but jquery-ui installs 3.7.0 (this can be seen in the lock file).
That means global `$` depends on the context.. we set some data to $ and then datepicker tries to get it but it's empty.
The solution is to tell jquery-ui to use the same version as we do, via module resolutions.